app/main/views/index.py

```diff
@@ -19,6 +19,18 @@ def landing(service_id, document_id):
     if not key:
         abort(404)
 
+    filename = request.args.get('filename')
+
+    return redirect(
+        download_link(
+            service_id,
+            document_id,
+            key,
+            filename
+        ),
+        code=302
+    )
```

Comment:
Redirect directly to the download link from the landing page. Do not touch other logic for now.